### PR TITLE
onCSSImportPath

### PR DESCRIPTION
**Pull request by octref**
Part of Microsoft/vscode#51331

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "editor.insertSpaces": false
```
Comment (octref) [inline .vscode/settings.json]: I did this because otherwise I always get hygiene problems.

**Comment by aeschli** [comment]
@octref Looks good.
Sorry I accidentally pushed Microsoft/vscode#51331, so pushing this one too.

**Comment by aeschli** [comment]
Don't forget to mention new APIs in the CHANGELOG. I added a entry for `ICompletionParticipant.onCssImportPath`

**Comment by octref** [comment]
> Don't forget to mention new APIs in the CHANGELOG.

Will do next time, thanks!

**Pull request merged**